src/scripts/domain/moveCalculator.ts

```
@@ -1,53 +1,109 @@
+      pos.add(dV);
+
+      cell = board.getCell(pos);
+    }
+  }
+
+  private static addKingMoves(
+    board: BoardInfo,
+    moves: Move[],
+    piece: PieceInfo
+  ) {
+    for (const direction of this.allDirections) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)